### PR TITLE
Use CNIVersion 0.3.1

### LINKED ISSUE #84: "IPv6 only" support for external CNIs and fakeipam
**Is this a BUG REPORT or FEATURE REQUEST?**:
bug

**What happened**:
If only IPv6 address is requested for an external CNI managed interface (e.g. sriov & macvlan), CNI returns with `invalid CIDR address: <nil>` error message.

**What you expected to happen**:
"IPv6 only" request should work.

**How to reproduce it**:
Only IPv6 address is requested for external CNI managed interface in DANM interface annotation:
```
  annotations:
    danm.k8s.io/interfaces: |
      [
        {"network":"flannel", "ip":"dynamic"},
        {"network":"sriov", "ip6":"dynamic"},
      ]
```

### PR DESCRIPTION
**Pull request by TothFerenc**
**What type of PR is this?**
bug

**What does this PR give to us**:
Allow to request only IPv6 address for external CNI managed interfaces. 

**Which issue(s) this PR fixes**:
Fixes #84 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
NONE

### CODE REVIEW

File: pkg/cnidel/cniconfs.go [hunk 1]
```diff
@@ -49,13 +47,13 @@ func getSriovCniConfig(netInfo *danmtypes.DanmNet, ipamOptions danmtypes.IpamCon
   if err != nil {
     return nil, errors.New("failed to get the name of the sriov PF for device "+ ep.Spec.Iface.DeviceID +" due to:" + err.Error())
   }
-  vlanid := netInfo.Spec.Options.Vlan
   sriovConfig := SriovNet {
+    CNIVersion: "0.3.1",
```
Comment (Levovar) [inline pkg/cnidel/cniconfs.go]: why are you hard-coding CNI version into the middle of the code, when it was already data structure driven?
the value should be just passed through the interface, now that we want to use it

File: pkg/cnidel/types.go [hunk 2]
```diff
@@ -15,6 +15,8 @@ type cniBackendConfig struct {
 
 // sriovNet represent the configuration of sriov plugin v1.0.0
 type SriovNet struct {
+  // CNI version of NetConf & Result structures
+  CNIVersion string `json:"cniVersion"`
```
[the remark this answers is not in the document]
Comment (TothFerenc) [reply pkg/cnidel/types.go]: `"github.com/containernetworking/cni/pkg/types".NetConf` has been embedded in the CNI specific structs, which provides the common CNI parameters.

File: pkg/cnidel/cniconfs.go (hunk 1 above)
Comment (TothFerenc) [reply pkg/cnidel/cniconfs.go]: done

**Comment by Levovar** [comment]
LGTM, will check out the UTs

**Pull request merged**